devfiles/dotnet-asp.net/devfile.yaml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: 1.0.0
+metadata:
+  name: asp.net
+projects:
+  -
+    name: aspnetcore-example
+    source:
+      type: git
+      location: "https://github.com/gothinkster/aspnetcore-realworld-example-app"
+components:
+  -
+    type: chePlugin
+    id: redhat-developer/che-omnisharp-plugin/latest
+    memoryLimit: 1024Mi
+  -
+    type: chePlugin
+    id: redhat-developer/netcoredbg-theia-plugin/latest
+    memoryLimit: 512Mi
+  -
+    type: dockerimage
+    alias: dotnet
+    image: mcr.microsoft.com/dotnet/core/sdk:2.2-stretch
```

Comment:
Before merging this PR, please update this to use one of the patched images (`quay.io/eclipse/che-dotnet-2.2:nightly`)